app/views/content_items/statistics_announcement.html.erb

```diff
@@ -9,15 +9,21 @@
 
 <div class="grid-row">
   <div class="column-two-thirds">
+    <%= render "components/important-metadata",
```

Comment:
Have you chatted to @miaallers about this change? I know when we moved things over to the universal layout, she wanted the metadata and notice tied together if they both exist on the same page. This change introduces a space between the two.

Reply:
Ok, from the ordering diagram it looked like there was a space between the two, but I'll check with her 👍

Reply:
Mia confirmed there should be no space between them, so I'll change that now 🚀